vsphere/cpi.yml

```diff
@@ -48,14 +48,22 @@
     password: ((vcenter_password))
     datacenters:
     - name: ((vcenter_dc))
-      vm_folder: ((vcenter_vms))
-      template_folder: ((vcenter_templates))
-      datastore_pattern: ((vcenter_ds))
-      persistent_datastore_pattern: ((vcenter_ds))
-      disk_path: ((vcenter_disks))
-      clusters:
-      - ((vcenter_cluster)): {}
+      vm_folder: ((vcenter_vm_folder))
```

Comment:
let's not rename bunch of variables since it's a bit invasive for not much in return at this time.